### ISSUE TITLE
Crashes on PgDown

### ISSUE THREAD

**Issue by aquilax** [bcicen/ctop#82]
The application crashes when I press the PageDown button:
```
$ ctop -v
ctop version 0.6.0, build 617b1b2
```
```go runtime error: slice bounds out of range

goroutine 239 [running]:
github.com/bcicen/ctop/cwidgets/compact.(*CompactGrid).Align(0xc4201b2e10)
	/home/bradley/go/src/github.com/bcicen/ctop/cwidgets/compact/grid.go:29 +0x28e
main.(*GridCursor).PgDown(0xc4201a4e40)
	/home/bradley/go/src/github.com/bcicen/ctop/cursor.go:180 +0x1c4
main.(*GridCursor).PgDown-fm()
	/home/bradley/go/src/github.com/bcicen/ctop/grid.go:88 +0x2a
main.HandleKeys.func1(0x95ce4d, 0x8, 0xc420298010, 0xf, 0x95c723, 0x7, 0x0, 0x0, 0x8d4720, 0xc420298020, ...)
	/home/bradley/go/src/github.com/bcicen/ctop/keys.go:39 +0x24
github.com/bcicen/ctop/vendor/github.com/gizak/termui.(*EvtStream).Loop.func1(0xc42001c730, 0x95ce4d, 0x8, 0xc420298010, 0xf, 0x95c723, 0x7, 0x0, 0x0, 0x8d4720, ...)
	/home/bradley/go/src/github.com/bcicen/ctop/vendor/github.com/gizak/termui/events.go:251 +0xff
created by github.com/bcicen/ctop/vendor/github.com/gizak/termui.(*EvtStream).Loop
	/home/bradley/go/src/github.com/bcicen/ctop/vendor/github.com/gizak/termui/events.go:253 +0x167
```

**Comment by bcicen** [comment]
@aquilax is there an associated error message before the stack trace output?

**Comment by aquilax** [comment]
@bcicen sorry, thought I've copied that too:

```
panic: runtime error: slice bounds out of range
```

**Comment by bcicen** [comment]
Thanks! I was able to replicate the error and have pushed a fix in https://github.com/bcicen/ctop/commit/b88c143914f9f3568d633f02a7d4d56961bc897e; this will be included with the next minor release

**Comment by bcicen** [comment]
fix has been included in [v0.6.1](https://github.com/bcicen/ctop/releases/tag/v0.6.1)
